tests/features/schema-generator/GH3261.test.ts

```diff
@@ -0,0 +1,46 @@
+import { Collection, Entity, ManyToOne, MikroORM, OneToMany, PrimaryKey, Property, Unique } from '@mikro-orm/core';
+
+@Entity()
+export class User {
+
+  @PrimaryKey()
+  _id!: number;
+
+
+  @Property({ type: 'boolean' })
+  email!: string;
+
+}
+
+
+let orm: MikroORM;
+
+beforeAll(async () => {
+  orm = await MikroORM.init({
+    entities: [User],
+    dbName: 'GH3216',
+    type: 'mongo',
+  });
+  await orm.getSchemaGenerator().dropSchema();
+});
+
+afterAll(() => orm.close(true));
+
+test('retry limit to 3 when ensureIndex() fails', async () => {
+  const user1 = await orm.em.create(User, {
+    email: 'test',
+  });
+  const user2 = await orm.em.create(User, {
+    email: 'test',
+  });
```

Comment:
`em.create` is sync method

```suggestion
  const user1 = orm.em.create(User, {
    email: 'test',
  });
  const user2 = orm.em.create(User, {
    email: 'test',
  });
```